modules/ROOT/pages/policies-policy-types.adoc

```diff
@@ -0,0 +1,49 @@
+= Policy Types
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies are available in different options, such as, Default, Custom, and Automated. Each type of the policy differs by its scope, management, and usability.
+
+== Default Policies
+
+Mulesoft provides several policies that are ready-to-use and cover different areas, such as authentication, security management, threat protection, and tokenization. You can apply any of these policies using the UI to any of your API endpoints. You can also use an API to apply policies.
+
+After you apply a policy, it is managed by the API Manager for its complete lifecycle. You can apply a Default policy to a specific API. To automatically apply a policy to to all APIs in your environment, you must configure it as  a policy.
+
+//It would actually be a good thing to add in the future, a section about CI and the link to the API API Manager portal. -- Eva?
+
+//[PA]: DO we release new versions of an existing policy? How does it affect a customer that already has a version applied? What do they have to do with it.
+
+== Automated Policies
```

Comment:
This is referred to earlier as "automatic." GLOBALLY standardize to the proper name.

Reply:
It should be automated, not automatic.

Reply:
It is automated everywhere. Done! Thanks!